### PR TITLE
update to separate from pagerduty environment variables

### PR DESCRIPTION
**Pull request by aaronblythe**
- Use separate defaults from the pager-me set up for webhooks
- Update README to favor user (click and labels)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,9 +9,10 @@ These are things you will need to acquire in order to run the CFN Template in AW
 ## Slack
 You will need to generate a slack token for hubot.
 
-1. Go to https://$yourteam.slack.com/services/new/hubot and create a new hubot integration
-2. Choose a bot username, this is what you will call to the bot in rooms as (e.g.; @hubot help)
-3. Find the API Token section (the token will start with `xoxb-`)
+1. [Hubot Slack App](https://slack.com/apps/A0F7XDU93-hubot)
```
Comment (TheFynx) [inline README.md]: why have both 1 and 2? Step two takes you directly to the create username for a new hubot. 

We need to keep only one and if it's step 1, then we need to add additional steps to cover the screens.

File: incident-bot.json [hunk 2]
```diff
@@ -406,7 +406,7 @@
                 "export PAGERDUTY_SERVICE_API_KEY=", { "Ref" : "ENVPagerDutyServiceKey" },"\n",
                 "export PAGERDUTY_SUBDOMAIN=", { "Ref" : "ENVPagerDutySubDomain" },"\n",
                 "export PAGERDUTY_USER_ID=", { "Ref" : "ENVPagerDutyUserID" },"\n",
-                "export PAGERDUTY_ROOM=", { "Ref" : "ENVPagerDutyRoom" },"\n",
+                "export HUBOT_INCIDENT_PAGERDUTY_ROOM=", { "Ref" : "ENVPagerDutyRoom" },"\n",
```
Comment (TheFynx) [inline incident-bot.json:409]: changing it here is only half the steps needed, you also have to modify the userdata.sh

File: README.md (hunk 1 above)
Comment (aaronblythe) [inline README.md]: Good point.  Do you prefer one or the other.  I like the clickable link.  I have seen these links on other tutorials however have no idea if they are permanent.

Comment (TheFynx) [inline README.md]: With this being the TL;DR generation, I prefer the simplest steps possible. So we can have a clickable link + 3-4 more steps or a non-clickable link with 3 steps. Either or is fine, but I see questions afoot.

File: incident-bot.json (hunk 2 above)
Comment (aaronblythe) [inline incident-bot.json:409]: https://github.com/HearstAT/cfn-incident-bot/pull/2/commits/55ef4211c6982c76e9330ff43c559c8195ce7ab4

File: README.md (hunk 1 above)
Comment (aaronblythe) [inline README.md]: Removed the added line with the link.

**Comment by TheFynx** [comment]
:+1: looks good should work without any modifications to the cookbook as well.

**Comment by aaronblythe** [comment]
For some reason I thought this was already set: `HUBOT_PAGERDUTY_USER_ID`  however looking through it now, I am not sure.

**Comment by aaronblythe** [comment]
I see it is `PAGERDUTY_USER_ID` and I should update this work to match that: https://github.com/HearstAT/hubot-incident/commit/c16cca1d034b0a068b0614cae02eb799d57adff3

This is good to merge.

**Pull request merged**